predict_loading_concentration.py

```diff
@@ -4,35 +4,37 @@ def warn(*args, **kwargs):
 import warnings
 warnings.warn = warn
 
-from diva_dna_seq.utility import aproximate_sequence,prepare_data
-from diva_dna_seq import predict_loading_concentration
+from diva_seq_opt import utility
 import argparse
 import pickle
 
 
+def main():
+	#Allow Matplotlib to be used on commandline
+	import matplotlib as mpl
```

Comment:
I think this should be outside of main()

Reply:
Great catch!